static/src/js/components/Tour/SearchTour.scss

```diff
@@ -0,0 +1,157 @@
+:root {
+  --color-white: #ffffff;
+  --color-black: #000000;
+  --color-heading: #0063A6;
+  --color-gray: #9a989a;
+  --color-light-gray: #f5f5f5;
+  --font-mono: "DM Mono", SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+}
+
+.tour {
+  &-heading {
+    font-family: var(--font-mono);
+    color: var(--color-heading);
+    font-size: 1.5rem;
+    font-weight: bold;
+    margin-bottom: 0.9375rem;
+    text-transform: uppercase;
+    letter-spacing: 0.0625rem;
+  }
+
+  &-subheading {
+    font-size: 1.5rem;
+    font-weight: bold;
+    margin-bottom: 1.25rem;
+  }
+
+  &-content, &-webinar-description {
+    font-size: 1rem;
+    margin-bottom: 1.25rem;
+    text-align: left;
+  }
+
+  &-note {
+    font-size: 0.875rem;
+    margin-bottom: 1.25rem;
+    text-align: left;
+  }
+
+  &-buttons {
+    display: flex;
+    justify-content: flex-end;
+    gap: 0.9375rem;
+    position: absolute;
+    bottom: 1.25rem;
+    right: 1.25rem;
+
+    &.intro {
+      width: 19rem;
+    }
+
+    &:not(.intro) {
+      width: 9rem;
+    }
+  }
+
+  &-info-box {
+    background-color: #f8f9fa;
+    padding: 1rem 1rem 0.0625rem;
+    margin-bottom: 0.9375rem;
+    border-radius: 0.25rem;
+    font-size: 0.875rem;
+    text-align: left;
+
+    a {
```

Comment:
Is there a reason we are using an `a` tag here rather than adding a classname to the links and giving them their own class? We should be avoiding nested block selectors as they greatly increase specificiity.